app/controllers/alaveteli_pro/stripe_webhooks_controller.rb

```diff
@@ -55,15 +55,22 @@ def check_for_event_type
       e = NoMethodError.new("undefined method `type' for " \
                             "#{@stripe_event.inspect}")
       # reject the webhook
-      notify_exception(e)
+      notify_exception(e, { webhook_data: @stripe_event} )
       render json: { error: e.message }, status: 400
       return
     end
   end
 
-  def notify_exception(error)
+  def notify_exception(error, data = nil)
     if send_exception_notifications?
-      ExceptionNotifier.notify_exception(error, :env => request.env)
+      if data
+        ExceptionNotifier.notify_exception(
+          error,
+          data: { webhook_data:  data }
```

Comment:
Quick Q: What's the reason for not including `request.env` in this case?

Reply:
oh, er - oversight I think

although I'm not convinced there'll be anything useful in `request.env` in this case (but as the whole point is to have as much information as possible about what might have gone wrong...)

Reply:
Okay – I'll give this an initial review but I don't see the point of adding extra conditionals here when we want the `request.env` anyway.

Reply:
agreed, will fix (simplify) post review - thanks!

rough sketch:

```diff
- def notify_exception(error, data = nil)
+ def notify_exception(error, data = {})
...
# remove conditional, always send `data` and `request.env` to notifier
```